doc/input-output-reference/src/input-for-output.tex

```diff
@@ -2381,3 +2381,65 @@ \subsubsection{Inputs}\label{outputcontrolfiles-inputs}
+
+Controls the format of the timestamp used in the native CSV and JSON outputs. The default timestamp is formated
+as ``$MM$/$DD$ $hh$:$mm$:$ss$'', where
+
+\begin{wherelist}
+\item[MM] the two-digit month, 
+\item[DD] the two-digit day of the month,
+\item[hh] the two-digit hours since the beginning of the day,
+\item[mm] the two-digit minutes since the beginning of the last hour, and
+\item[ss] the two-digit seconds (which is always ``00'').
+\end{wherelist}
+
+The time portion of this timestamp has traditionally been the elapsed time since the beginning of the day and associates values
+with the elapsed time at the end of the interval of interest.  For example, the timestamp for the interval from 1~AM to 2~AM on
+December 7 is written as ``12/07 02:00:00''. Unfortunately, some common software used to postprocess results has trouble
+interpreting the last timestamp for each day, which will include ``24:00:00''. An alternative that avoids this problem is to
+associate values with the elapsed time at the beginning of the interval of interest. In this case, the interval from 1~AM to
+2~AM on December 7 is written as ``12/07 01:00:00'' and 24 will never appear as the hours value (but 00 will). The ISO 8601
+timestamp format is ``$YYYY$-$MM$-$DD$T$hh$:$mm$:$ss$'' where
+
+\begin{wherelist}
+\item[YYYY] the four-digit year,
+\item[MM] the two-digit month, 
+\item[DD] the two-digit day of the month,
+\item[hh] the two-digit hours since the beginning of the day,
+\item[mm] the two-digit minutes since the beginning of the last hour,
+\item[ss] the two-digit seconds (which is always ``00''),
+\end{wherelist}
+
+and a capital letter ``T'' separates the time portion of the timestamp from the date portion. The timestamp for the interval from
+1~AM to 2~AM on December 7, 2022 is written either as
+
+\begin{itemize}
+\item ``2022-12-07T02:00:00'' at the end of the interval or
+\item ``2022-12-07T01:00:00'' at the beginning of the interval.
+\end{itemize}
+
+Output workflows using the ReadVars postprocessing program will not be impacted by the selections made here.
+
+\subsubsection{Inputs}\label{inputs-for-timestamp}
+
+\paragraph{Field: ISO 8601 Format}\label{field-timestamp-iso8601format}
+
+This field determines whether the ISO 8601 or traditional format is used. If the field is ``\textbf{No}'', the traditional format is
+used. If the field is ``\textbf{Yes}'', the ISO 8601 format is used. The default is ``\textbf{No}''.
+
+\paragraph{Field: Timestamp at Beginning of Interval}\label{field-timestamp-beginning}
+
+This field determines whether the timestamp is placed at the beginning of the interval (``\textbf{Yes}'') or at the end of the
+interval (``\textbf{No}''). The default is ``\textbf{No}''.
+
+\subsubsection{Example}\label{example-for-timestamp}
+
+An example IDF input object that uses the ISO 8601 format and places the timestamp at beginning of the interval.
+\begin{lstlisting}
+Output:SQLite,
```

Comment:
@jasondegraw This should be `OutputControl:Timestamp` not `Output:SQLite`, correct?

I came across this because I was wondering if non-CSV outputs (SQLite, JSON) are affected by this new object and I can't tell based on the description. Can you clarify?

Reply:
(By the way, thanks so much for working on this and incorporating the idea of the begin/end of interval convention.)

Reply:
@shorowit Yes, that should be `OutputControl:Timestamp`. I noticed another typo that will need to be fixed, so I'll correct that today.

Really not being sarcastic (I know that's hard to believe), but I had hoped that the first sentence of the object description was sufficient: "Controls the format of the timestamp used in the native CSV and JSON outputs." Any suggestions on making things clearer?

Reply:
Thanks, @jasondegraw. You're right, it is pretty clear. I guess I hoped that SQLite might have come along for the ride for free, if all the formats referenced the same timestamp code; wishful thinking!

Reply:
Forgot to add this, but in discussion with other team members, the general plan is to expand the outputs that this object effects in future releases. Outputs that are tightly connected to the ESO are less likely to change. For one thing, there isn't really a timestamp in the ESO, just the pieces of a timestamp. The main change there would be moving to the beginning of the interval and making sure there's always a year, and I don't know if that is possible. Maybe? The tabular outputs that already have a non-ISO 8601 timestamp are likely first targets.

Just saw your response, and unfortunately SQLite falls into the "tightly connected to ESO" category as far as I know. But I could be wrong, so it wouldn't hurt to put in an issue specifically about SQLite. There's already #9272, which is related but is not as specific.